substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -580,6 +583,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type ChillThreshold<T: Config> = StorageValue<_, Percent, OptionQuery>;
 
+	/// The percentage of the reward inflation that is minted directly into the treasury.
+	#[pallet::storage]
+	pub type TreasuryInflationFraction<T: Config> = StorageValue<_, Percent, OptionQuery>;
```

Comment:
With `ValueQuery` you get the default value which should be `Zero::zero()` all the time.